filters/hsl-adjustment/src/HslAdjustmentFilter.ts

```diff
@@ -25,13 +25,26 @@ class HslAdjustmentFilter extends Filter
 
     /** Default values for options. */
     static readonly defaults: HslAdjustmentFilterOptions = {
+        /** Hue */
         hue: 0,
+        /** Saturation */
         saturation: 0,
+        /** Lightness */
         lightness: 0,
+        /** Colorize */
         colorize: false,
+        /** Alpha */
         alpha: 1,
```

Comment:
I don't think these comments add any value. However if it aligns with the other filters I guess it make sense to have them.